plutus-core-spec/plutus-core-specification.tex

```diff
@@ -268,14 +276,11 @@ \subsection{Blockchain issues}
 time and requires memory, so there is also a gas charge for on-chain
 type checking.
 
-%% Because of these issues, it is important that the cost of on-chain
-%% code execution is easy to calculate, certainly during execution (so
-%% that gas consumption can be accounted for), but ideally also in
-%% advance so that a user has some idea of how much a contract is likely
-%% to cost and can make sure that they buy enough gas to allow their
-%% contract to run to completion.  The design of Plutus Core incorporates
-%% a number of features to facilitate cost analysis for this purpose.
-
+Plutus Core is intended to be used for validation of on-chain
+transactions on the Cardano blockchain~\citep{Cardano}.  The details
+of the validation process are outside the scope of this document, but
+are explained in~\citep{EUTXO}. See also~\citep{Plutus-book} for 
+details of Cardano's smart contract infrastructure.
```

Comment:
I probably wouldn't cite the plutus book, since it's not really supposed to be a *technical* reference.